packages/beeq/src/components/date-picker/bq-date-picker.tsx

```diff
@@ -485,7 +485,7 @@ export class BqDatePicker {
         </label>
         {/* Select date picker dropdown */}
         <bq-dropdown
-          class="bq-date-picker__dropdown w-full [&::part(panel)]:w-auto [&::part(panel)]:p-m"
+          class="bq-date-picker__dropdown is-full p-b-m [&::part(panel)]:is-auto [&::part(panel)]:p-i-m"
```

Comment:
It seems that the addition of `p-b-m` is a mistake, maybe the intention was to set:
```suggestion
          class="bq-date-picker__dropdown is-full [&::part(panel)]:is-auto [&::part(panel)]:p-b-m [&::part(panel)]:p-i-m"
```
but if the padding remains the same all over, then `p-m` should be enough, no need to split in padding-block and padding-inline ;-)

Reply:
I pushed a fix for it!